Sources/FirebaseAuth/FirebaseAuth+Swift.swift

```diff
@@ -87,7 +87,14 @@ extension Auth {
       }
       future.Wait(firebase.FutureBase.kWaitTimeoutInfinite)
     }
-    return providers.pointee.providers.compactMap(String.init)
+
+    // Workaround WIN-971 by not using compactMap which uses std.vector<std.string>::const_iterator.
+    // return providers.pointee.providers.compactMap(String.init)
+    var result: [String] = []
+    for idx in 0..<providers.pointee.providers.size() {
+      result.append(String(providers.pointee.providers[idx]))
+    }
+    return result
```

Comment:
It would be nice to keep both paths with a SR flag to track the issue.

```suggestion
#if SR?????
    // Workaround WIN-971 by not using compactMap which uses std.vector<std.string>::const_iterator.
    var result: [String] = []
    for idx in 0 ..< providers.pointee.providers.size() {
      result.append(String(providers.pointee.providers[idx]))
    }
    return result
#else
    return providers.pointee.providers.compactMap(String.init)
#endif
```

Reply:
We can tweak the Package.swift to define the SR (upstream issue) that is associated with the issue.

Reply:
I assume upstream here is apple/swift. Do I need a reduced test / repro steps to get the SR number? That might not be instantaneous.

Reply:
Lets file the issue and indicate that you will provide a reduced test case as a follow up.  That way we can get this in and then you can work on the reduced test case.

Reply:
Done